docs/src/main/paradox/scala/http/routing-dsl/directives/form-field-directives/index.md

```diff
@@ -5,9 +5,9 @@
 @@@ index
 
 * [formField](formField.md)
-* [formFields](formFields.md)
+* @scala[[formFields](formFields.md)]
```

Comment:
does this look fine or shows empty `*` in java?

Reply:
The link `FormFields` doesn't appear in the Java version

Reply:
I know the *link* won't, I'm worried if an empty:

- formField
-
- other thing

will appear for java

Reply:
This is how it appears, no empty space :)
![selection_006](https://user-images.githubusercontent.com/3337739/34300913-a9190754-e72a-11e7-949c-5e09464e241c.jpg)